ext/TensorKitChainRulesCoreExt.jl

```diff
@@ -385,14 +385,17 @@ function svd_pullback!(ΔA::AbstractMatrix, U::AbstractMatrix, S::AbstractVector
 
     # check whether cotangents arise from gauge-invariance objective function
     mask = abs.(Sp' .- Sp) .< tol
-    gaugepart = view(aUΔU, mask) + view(aVΔV, mask)
-    norm(gaugepart, Inf) < tol || @warn "cotangents sensitive to gauge choice"
+    gaugepart = norm(view(aUΔU, mask) + view(aVΔV, mask), Inf)
+    gaugepart < tol ||
+        @warn "`svd` cotangents sensitive to gauge choice: (|ΔGauge| = $gaugepart)"
     if p > r
         rprange = (r + 1):p
-        norm(view(aUΔU, rprange, rprange), Inf) < tol ||
-            @warn "cotangents sensitive to gauge choice"
-        norm(view(aVΔV, rprange, rprange), Inf) < tol ||
-            @warn "cotangents sensitive to gauge choice"
+        aUΔUpart_norm = norm(view(aUΔU, rprange, rprange), Inf)
+        aUΔUpart_norm < tol ||
+            @warn "`svd` cotangents sensitive to gauge choice: (|aUΔU| = $aUΔUpart_norm)"
+        aVΔVpart_norm = norm(view(aVΔV, rprange, rprange), Inf)
+        aVΔVpart_norm < tol ||
+            @warn "`svd` cotangents sensitive to gauge choice: (|aVΔV| = $aVΔVpart_norm)"
     end
```

Comment:
Maybe remove the warnings above, have a single warning below the end and replace the `aUΔUpart_norm` and `aVΔVpart_norm` parts, i.e. as follows:
```
gaugepart = norm(view(aUΔU, mask) + view(aVΔV, mask), Inf)
if p > r
  rprange = (r + 1):p
  gaugepart = max(gaugepart, norm(view(aUΔU, rprange, rprange), Inf))
  gaugepart = max(gaugepart, norm(view(aVΔV, rprange, rprange), Inf))
end
gaugepart < tol ||
         @warn "`svd` cotangents sensitive to gauge choice: (|ΔGauge| = $gaugepart)"
```